requirements.txt

```diff
@@ -0,0 +1,5 @@
+asgiref==3.6.0
+Django==4.1.5
+python-dotenv==0.21.0
+sqlparse==0.4.3
+tzdata==2022.7
```

Comment:
nit-picking, to było zrobione przez kopię pip freeze, ja preferuję jednak dodanie tych paczek które instalujemy przez pip install, czyli w tym wypadku `python-dotenv` i `Django`